<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LD2410B Configuration Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEditMessagesLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>461</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSerial">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenPort">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>by Miguel Moreto - 2024</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSendCmd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendCmd">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>160</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Command</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBoud">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string>256000</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnSerialRefresh">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Refresh the list of serial ports.</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../pyLD2410B/images/images.qrc">
      <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="labelbytes">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeltmp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClosePort">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Port</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbout"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../pyLD2410B/images/images.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close the window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../pyLD2410B/images/images.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Show about screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../pyLD2410B/images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
